package.json

```diff
@@ -11,15 +11,15 @@
   "dependencies": {
     "axios": "^0.27.2",
     "core-js": "^3.6.5",
+    "formik": "^2.2.9",
     "js-cookie": "^3.0.1",
+    "lodash": "^4.17.21",
     "react": "18.1.0",
+    "react-copy-to-clipboard": "^5.1.0",
     "react-dom": "18.1.0",
     "react-query": "^3.39.1",
-    "react-router-dom": "^6.3.0",
-    "formik": "^2.2.9",
-    "lodash": "^4.17.21",
-    "react-copy-to-clipboard": "^5.1.0",
     "react-resize-detector": "^7.1.2",
+    "react-router-dom": "^6.3.0",
```

Comment:
❓ This happens after `npm install` in root. I can commit in a separate PR instead. Shall I?